relnotes/ipool.feature.md

```diff
@@ -0,0 +1,7 @@
+* `ocean.util.container.pool.model.IPool`, `ocean.util.container.pool.model.IAggregatePool`
+
+  A new abstract function `isBusy()` has been added to the API of IPool. Classes
+  wishing to inherit from `IPool` must provide an implementation for it.
```

Comment:
Why don't you move this method to `IPool` itself, possibly even marking as final? Both `num_busy` and `getItemIndex` are available in it.